src/components/timeline/ActiveTabResourceTrack.js

```diff
@@ -147,7 +147,7 @@ class ActiveTabResourceTrackComponent extends PureComponent<Props, State> {
     let trackLabel;
     switch (resourceTrack.type) {
       case 'sub-frame':
-        trackLabel = 'Frame:';
+        trackLabel = 'Iframe:';
```

Comment:
I think `Iframe` is a little weird with the capitalization. Technically this is `HTMLIFrameElement` I would either change this to `iframe` or `IFrame`, or `<iframe>`.

Reply:
Yeah, I wasn't completely sure about that but saw some "Iframe" capitalization and initially did that. I feel like `IFrame` is better for this case then, `<iframe>` would be more explicit but it doesn't look here when I change it. Changed it to `IFrame`.